### PR TITLE
[v7r2] Hackathon fixes: logging and tests

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES
*Framework
FIX: MQ logging sends also the message,asctime and levelname field

ENDRELEASENOTES

### CODE REVIEW

File: Resources/LogBackends/MessageQueueBackend.py [hunk 1]
```diff
@@ -32,6 +35,14 @@ def __init__(self, backendParams=None):
     """
     Initialization of the MessageQueueBackend
     """
+    # The `Format` parameter is passed as `fmt` to libJsonFormatter
+    # which uses it to know which "standard" fields to keep in the
+    # json output. So we need these
+    if not backendParams:
+      backendParams = {'Format': DEFAULT_FMT}
+    elif 'Format' not in backendParams:
+      backendParams['Format'] = DEFAULT_FMT
```
Comment (chrisburr) [inline Resources/LogBackends/MessageQueueBackend.py:44]: Rather that duplicating, how about:
```suggestion
    if backendParams is None:
      backendParams = {}
    if 'Format' not in backendParams:
      backendParams['Format'] = DEFAULT_FMT
```
or more concisely:
```suggestion
    if backendParams is None:
      backendParams = {}
    backendParams.setdefault("Format", DEFAULT_FMT)
```

File: FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py [hunk 2]
```diff
@@ -37,6 +37,9 @@ def __init__(self, queue):
     result = createProducer(queue)
     if result['OK']:
       self.producer = result['Value']
+    else:
+      # print because logging not available
+      print("ERROR initializing MessageQueueHandler: %s" % result)
```
Comment (chrisburr) [inline FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py:42]:
```suggestion
      print("ERROR initializing MessageQueueHandler:", result)
```

Comment (chaen) [reply FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py:42]: What's wrong with the `%s` ?

File: Resources/LogBackends/MessageQueueBackend.py (hunk 1 above)
Comment (chaen) [reply Resources/LogBackends/MessageQueueBackend.py:44]: Good idea

File: FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py (hunk 2 above)
Comment (chrisburr) [reply FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py:42]: I think it's better style to use multiple arguments to `print` as `%` formatting is longer, harder to lint and more likely to result in an exception due to accidentally passing the wrong number of parameters.

That said obviously in this case it makes no difference as I'm 99% sure it's correct as-is so I'm happy either way.

Comment (chaen) [reply FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py:42]: yeah, the multitude of way to format string does not really help... between the C-style, the multiple number of arguments, and the `.format`....

**Review by aldbr — APPROVED** [review]

File: FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py (hunk 2 above)
Comment (chrisburr) [reply FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py:42]: You missed the one true way! (`f"Hello {name}"`) 😆

**Review by chrisburr — APPROVED** [review]

File: FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py (hunk 2 above)
Comment (chaen) [reply FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py:42]: True ! :D

**Review by atsareg — APPROVED** [review]

**Pull request merged**